### ISSUE TITLE
Clientconfig used instead serverconfig

### ISSUE THREAD

**Issue by Thyriel** [MikeLydeamore/ExNihiloAdscensio#107]
I'd guess this is more an enhancement request and not a bug, but imho it's very unusual that a mod loads the client config when playing on a server.
Would be great if the client loads the configs from the server so i don't need to distribute changes in the server config files from ex nihilo manually to every player.
Background: I have completely rewritten the sieve drops, but clients still display the default drops in JEI. (But drops work like in server config)

**Comment by MikeLydeamore** [comment]
This isn't unusual at all - almost every config differs on client and server. Take Minetweaker for example - if the server has an extra script to add a recipe, it won't show on the client, but it will be craftable.

It's also a very challenging problem to solve. Some configs change the presence of blocks or items, which requires a reload of Minecraft (not just the world, the entire client). JEI recipes are cached at load time now, so those also would need to be re-run.

In short, while I understand that it might seem counterintuitive, it's not going to get changed.
